### PR TITLE
Modifies unit tests for the new FormatterResolver

### PR DESCRIPTION
**Pull request by binaek**

### CODE REVIEW

File: pkg/db/db_common/mock_client.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import (
+	"context"
+	"database/sql"
+
+	"github.com/turbot/steampipe/pkg/query/queryresult"
+	"github.com/turbot/steampipe/pkg/schema"
+	"github.com/turbot/steampipe/pkg/steampipeconfig"
+)
+
+type MockClient struct{}
```
Comment (kaidaguerre) [inline pkg/db/db_common/mock_client.go:12]: what is this for?

File: pkg/export/manager.go [hunk 2]
```diff
@@ -3,10 +3,11 @@ package export
 import (
 	"context"
 	"fmt"
-	"golang.org/x/exp/maps"
 	"path"
 	"strings"
 
+	"golang.org/x/exp/maps"
+
```
Comment (kaidaguerre) [inline pkg/export/manager.go:10]: remove newline

File: pkg/control/controldisplay/formatter_test.go [hunk 3]
```diff
@@ -1,263 +1,125 @@
...
-// 				}
-// 			]
-// 		}
-// 	],
-// 	"controls": null
-// }`
+var formatterTestCase = []testCase{
+	{
+		input:    "bad-format",
+		expected: "ERROR",
```
Comment (kaidaguerre) [inline pkg/control/controldisplay/formatter_test.go:37]: do we have test cases for alias?

File: pkg/export/manager_test.go [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+		input:  "file.json",
+		expect: &dummyJSONExporter,
+	},
+	{
+		name:   "ASFF JSON",
+		input:  "file.asff.json",
+		expect: &dummyASFFExporter,
+	},
+	{
+		name:   "NUNIT3",
```
Comment (kaidaguerre) [inline pkg/export/manager_test.go:62]: no need for all caps

File: pkg/export/manager_test.go [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+type exporterTestCase struct {
+	name   string
+	input  string
+	expect interface{}
+}
+
+var exporterTestCases = []exporterTestCase{
+	{
+		name:   "Bad Format",
```
Comment (kaidaguerre) [inline pkg/export/manager_test.go:37]: lets also have some format name tests here (as well as extension)

File: pkg/export/manager_test.go [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+	m := NewManager()
+	for _, e := range exportersToRegister {
+		m.Register(e)
+	}
+	for _, testCase := range exporterTestCases {
+		targets, err := m.resolveTargetsFromArgs([]string{testCase.input}, "dummy_execution_name")
+		shouldError := testCase.expect == "ERROR"
+		if shouldError {
+			if err == nil {
+				t.Logf("Request for '%s' should have errored - but did not", testCase.input)
```
Comment (kaidaguerre) [inline pkg/export/manager_test.go:86]: just use t.Errorf

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: pkg/control/controldisplay/formatter_test.go (hunk 3 above)
Comment (binaek) [reply pkg/control/controldisplay/formatter_test.go:37]: yes -  we have three dummy exporters (out of 5) which have an alias and is being tested.

File: pkg/export/manager_test.go (hunk 4 above)
Comment (binaek) [reply pkg/export/manager_test.go]: done

File: pkg/export/manager_test.go (hunk 5 above)
Comment (binaek) [reply pkg/export/manager_test.go:37]: added a few tests

File: pkg/export/manager_test.go (hunk 6 above)
Comment (binaek) [reply pkg/export/manager_test.go]: done

File: pkg/db/db_common/mock_client.go (hunk 1 above)
Comment (binaek) [reply pkg/db/db_common/mock_client.go]: removed it

**Pull request merged**